src/cmd/ksh93/data/msg.c

```diff
@@ -76,6 +76,7 @@ const char e_access[] = "permission denied";
 const char e_direct[] = "bad directory";
 const char e_file[] = "%s: bad file unit number";
 const char e_redirect[] = "redirection failed";
+const char e_io[] = "I/O error";
```

Comment:
Can this message be made more descriptive to help the user ? Is it only shown on some specific type of I/O error ?

Reply:
It is shown when `sfsync()` returns an error exit status. The sfio.3 error page says `sfsync()` "returns a negative value for failure and 0 for success." It doesn't say how to get more specific error information out of it, so I have no idea how to make the error more specific.